ci/tests/scaffolding/e2e-helm-scaffold-hybrid.sh

```diff
@@ -6,7 +6,9 @@ set -eux
 
 ROOTDIR="$(pwd)"
 HELMDIR="/go/src/github.com/helm-op"
+
 mkdir -p $HELMDIR
+cd $HELMDIR
```

Comment:
I believe that it will solve the issue. 
Also keeping it as it is done for Ansible.